c7n/resources/efs.py

```diff
@@ -28,6 +28,21 @@ class resource_type(object):
         date = 'CreationTime'
         dimension = None
 
+    def augment(self, filesystems):
```

Comment:
this code is extraneous, to just defining a details_spec on the resource_type

ie. worth trying this one liner, 
```
detail_spec = ("describe_tags", "FileSystemId", "FileSystemId", None) 
```